dbms/src/VectorSearch/USearch.h

```diff
@@ -23,17 +23,18 @@
 
 // Force enable all target features.
 #if defined(__APPLE__)
-#define SIMSIMD_TARGET_NEON 0
 #define SIMSIMD_TARGET_SVE 0
 #else
-#define SIMSIMD_TARGET_NEON 1
 #define SIMSIMD_TARGET_SVE 1
 #endif
```

Comment:
This check can be removed according to https://github.com/tidbcloud/tiflash-cse/pull/377